man/machinery-inspect-docker.1.md

```diff
@@ -0,0 +1,111 @@
+
+## inspect — Inspect Running System
+
+### SYNOPSIS
+
+`machinery inspect-container` [OPTIONS] IMAGENAME / IMAGEID
```

Comment:
Splitting it up in two lines could reduce the confusion if you are supposed to type a `/` in the command line:
`machinery inspect-container` [OPTIONS] IMAGENAME`
`machinery inspect-container` [OPTIONS] IMAGEID`